### PR TITLE
fix/return cannot send email error

### PR DESCRIPTION
**Pull request by markkovari**
- **refactor: protectedpage on frontend**
- **fix: return with dedicated errors and messages**

### CODE REVIEW

File: .env
Comment (AnnaUgrai) [inline .env:1]: Do we push the .env file here?
I thought it is in the gitignore 🗡️

Comment (markkovari) [reply .env:1]: well yes, mea culpa, but we should've added this years ago 😠

Comment (markkovari) [reply .env:1]: let me delete it as well

**Review by ramesz6 — APPROVED** [review]

**Comment by markkovari** [comment]
@gfa-cc-after/gilmore-devs 🥹

**Review by AnnaUgrai — APPROVED** [review]
image

**Pull request merged**